bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/IFindReplaceLogic.java

```diff
@@ -166,4 +166,15 @@ public interface IFindReplaceLogic {
 	 */
 	public IFindReplaceTarget getTarget();
 
+	/**
+	 * Returns <code>true</code> if searching should be restricted to entire words,
```

Comment:
```suggestion
	 * Returns <code>true</code> if searching can be restricted to entire words,
```

Reply:
Addressed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/a0b224321663e63aa99f60da4a2527f993d68f7d